architecture/introduction.mdx

```diff
@@ -0,0 +1,32 @@
+---
+title: "Introduction to Novu"
+description: "Overview of Novu's Architecture"
+---
+Novu is an open-source notification infrastructure designed to provide a robust, efficient, and flexible solution for managing notifications across various platforms.
+The architecture of Novu is meticulously crafted to ensure accurate delivery of notifications while maintaining high performance and scalability.
+
+Our systems are hosted on Amazon Web Services (AWS), ensuring a reliable and secure environment for our infrastructure.
+We adhere to several industry-standard compliance certifications including SOC II, PCI, and GDPR which underline our commitment to security and data privacy.
+
+## Framework Leveraged
+
+- Nestjs: A progressive Node.js framework that is used for building efficient, reliable, and scalable server-side applications. Nestjs provides a high-level abstraction with a lot of out-of-the-box features, making it a perfect choice for building modular and maintainable applications.
+- Mongoose: An elegant Object Document Mapper (ODM) for MongoDB, which simplifies the interaction with the database by providing a straightforward modeling of the application data. Mongoose’s features like schema validation, query building, and middleware support immensely help in building robust applications.
+- BullMQ: A robust, fast, and reliable queue library for handling distributed jobs and messages in Node.js. It is instrumental in managing the processing of notification jobs ensuring they are handled in an efficient and reliable manner.
```

Comment:
Please make `Nestjs`, `Mongoose`, `BullMQ` bold so it's differentiated from the detailed explanation of each